data/countries.txt

```diff
@@ -120,7 +116,8 @@ Mauritania
 Mauritius
 Mayotte
 Mexico
-Micronesia,Federated States of Micronesia
+Micronesia
+Federated States of Micronesia

```

Comment:
Why two entries?

Reply:
Yes they are now to seperate states according to UN data everything is based on doc links i provided up.

Reply:
then there might be ambiguity regarding keeping that in same line though there flags are same.
same goes with turkey/turkiye